FD 1.14.0 Reuse Config (#120)
mulesoft/docs-design-center: modules/ROOT/pages/fd-reuse-config.adoc

> 

Reviewer: INSTRUCTIONS FOR EDIT

can you review all the green on this PR? You already reviewed it, but a typo was discovered so while I had the file open, I made a couple of edits.  It's not a super-rush, but when you can would be great

:imagesdir: ../assets/images

> You can reuse a connection in a Flow Designer project instead of manually adding your authentication information.

Reviewer: How do you feel about putting the benefit right up front, something like "To save time, you can..." or "For improved efficiency, you can..."? Just a thought.

Reviewer: Another idea: "...instead of manually adding your authentication information" > "...instead of repeatedly reauthenticating."

Reviewer: ...but if you alter the text, please BE SURE TO REREAD what you have touched. 
It is a cruel reality that in correcting one mistake, we often make some typing error that creates another....

Author: Going to get the typos fixes in. Will circle back on these later.

After a reusable connection is created, it is available for all Flow Designer projects in your business group.
You can change your reusable configuration at anytime.

> You can reuse connections:

Reviewer: ```suggestion
You can reuse connections in the following circumstances:
```
I think we need this addition because the list items do not naturally flow, grammatically, from the shorter intro.

Author: Fixed.

> * If you are a system administrator, you can enable non-administrators to connect to your system using your authentication.

Reviewer: "authentication" > "credentials"?

Reviewer: Note that the step at Line 28, which is hidden in my copy of this file and cannot be edited directly, violates the one-sentence-per-list-item style guideline. I suggest reformatting it as follows:

. To make this connection available for users in your business group, leave the *share this connection with my business group* checkbox selected. 
To maintain security, only you as the creator of a shared connection can view your authentication credentials.

Author: The rest of the FD doc says authentication. That's why I kept it. I would have to change it globally.

. Choose a connector and operation.

> . View the list of your reusable connections. To help you choose the right connection, the creation date of the reusable connector is listed.

Reviewer: DITTO re. list items not exceeding one sentence:
```suggestion
. View the list of reusable connections. 
To help you choose the right connection, each reusable connector is accompanied by its creation date.
```
